tests/metrics/density/docker_cpu_usage.sh.in

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test checks the CPU usage of several idle containers and
+#  containers that are working running shuf.
+#  The measure is take after the workload has become stable.
+#  By default this tests run 100 containers and waits 2m to stabilize the
+#  workloads.
+
+set -e
+
+[ $# -ne 2 ] && ( echo >&2 "Usage: $0 <times to run> <wait time>"; exit 1 )
```

Comment:
hey... in every line like this you need to finish it with 

`| ::`

if you do not this it can lead to false positives.... 

ie:

```
$ [ 2 -eq 3 ] && ( echo foo; exit 1 )
gorozco1@gorozco1-mac01:~/Desktop
$ echo $?

```1